web/impact/impact/tests/test_utils.py

```diff
@@ -1,22 +1,28 @@
 # MIT License
 # Copyright (c) 2017 MassChallenge, Inc.
 
-from test_plus.test import TestCase
-from impact.utils import get_profile
-from impact.tests.contexts import (
-    UserContext,
-)
-from impact.models import (
-    ExpertProfile,
-    MemberProfile,
+from impact.models.utils import (
+    model_name_to_snake,
+    snake_to_model_name,
 )
+from impact.tests.api_test_case import APITestCase
+
+
+empty_string = ''
+model_name = 'ProgramStartupStatus'
```

Comment:
I would prefer if this were named `CAMEL_CASE_MODEL_NAME` (since it's a constant, and we're differentiating between different cases)

Reply:
Changed.